### PR TITLE
Fix: disable updateNetwork and updateGeoData on currentNode renaming

### PR DESCRIPTION
**Pull request by sBouzols**
Signed-off-by: sBouzols <sylvain.bouzols@gmail.com>

### CODE REVIEW

**Review by bendaamerahm — APPROVED** [review]
test ok 
=> that resolve the actual problem but we need to figure out a better solution later (handle specific Notification by action)

**Review by bendaamerahm — APPROVED** [review]
Code review: OK
Test: 
- renaming node don't trigger fetch of GeoData and Network : OK

File: src/components/graph/util/model-functions.js [hunk 1]
```diff
@@ -74,6 +74,18 @@ export function isNodeReadOnly(node) {
 }
 
 export function isNodeBuilt(node) {
+    if (!node) return false;
     if (node?.type === 'ROOT') return true;
```
Comment (laurgarn) [inline src/components/graph/util/model-functions.js:78]: You could get rid of the ? in the ?. due to the test you added above

File: src/components/study-container.js [hunk 2]
```diff
@@ -317,20 +322,18 @@ export function StudyContainer({ view, onChangeTab }) {
...
-                            : getFirstNodeOfType(tree, 'ROOT')
-                    )
-                );
-
+                let firstSelectedNode =
+                    getFirstNodeOfType(tree, 'NETWORK_MODIFICATION', 'BUILT') ||
+                    getFirstNodeOfType(tree, 'ROOT');
+
+                // To get positions we must get the node from the model class
+                let ModelFirstSelectedNode = {
```
Comment (laurgarn) [inline src/components/study-container.js:330]: Why let and not const ? (can not remember a language where constness cost less typing than mutability, though :-/ )

**Review by laurgarn — APPROVED** [review]

**Pull request merged**